File: problems/3512-minimum-operations-to-make-array-sum-divisible-by-k/solution.c
int minOperations(int* nums, int numsSize, int k) {
    int sum = 0;
    for (int i = 0; i < numsSize; i++) {
        sum += nums[i];
    }

    int remainder = sum % k;
    if (remainder == 0) {
        return 0;
    }

    return remainder;
}

File: problems/3512-minimum-operations-to-make-array-sum-divisible-by-k/solution.java
class Solution {
    public int minOperations(int[] nums, int k) {
        int sum = 0;
        for (int num : nums) {
            sum = (sum + num) % k;
        }
        if (sum == 0) {
            return 0;
        }
        return k - sum;
    }
}